### PR TITLE
added_readme_for_new_contributors

### LINKED ISSUE #112: Create a step by step docs for new contributors
Many new contributors are unable to submit their first PR due to a lack of info. I want a document that a new contributor can follow exactly to submit his/her first PR.

I want this documentation to be of high quality so, do not take this issue for the sake of points in GSSoC. 

**Information that needs to be present**
1. Introduction in the discord group
2. Find a new issue 
3. Get help from the community
4. Fork the repo
5. Make your changes
6. Push your changes to the forked repo
7. Create your first pull request
8. Get feedback from the mentors
9. Go to step 5 again

### PR DESCRIPTION
**Pull request by Sukriti-sood**
Created a step by step docs for new contributors

Fixes #112

I have Added Documentation for New Contributors . 

Currently I have not added Link of this new markdown in the Readme .

### CODE REVIEW

**Comment by priyanshuone6** [comment]
@Sukriti-sood Could we use numbered list instead of bullet points

File: docs/newContributors.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+<br>
+
+- ##### Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+    > **Open Source is more about Collaboration.**
+
+- ##### So to start Contibuting to our Project you can Solve existing issues or if you have found any bug or  want to add new featurre to Project you can create New Issue , that's purely your choice.
+  ![](../images/Issues.png)
+
+- ##### Project Maintainers and Mentors Will Guide you With the issue and Incase of any problem you face ,you can take help from the community in the discord server.
```
Comment (priyanshuone6) [inline docs/newContributors.md:14]: You have used capital letters in the middle of the document like in this line. Could you make them small and use them only to emphasise something.

**Review by priyanshuone6 — CHANGES_REQUESTED** [review]

File: docs/newContributors.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - ``git push origin branch_name``
+
+- ##### Go to your Forked Repo in Github , there you will see notification for Compare and Pull Request like this:-![](../images/github-comparepr.png) Click this and Create your First Pull Request.
+
+- ##### Mentors will review your changes and Will give give feedback and if everything Go well , your PR will get merged , Hurray 🎉🥳
+
+- ##### So Now Your PR Got Merged 🎉, Again Find or Create Issue and Continue the loop of Contributions to this Project🥳
+
+
+### Happy Coding 💻
```
Comment (priyanshuone6) [inline docs/newContributors.md:44]: Centering this line will make it look nice

**Review by priyanshuone6 — CHANGES_REQUESTED** [review]

File: docs/newContributors.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  
+- ##### So, Now You have an Issue to work upon. So First you need to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way.
+  ![](../images/Fork.png)
+
+- ##### Clone the Repo you have forked :-
+     - Copy the link of the forked Repo.
+     - Go to the desired location in the terminal, where you want to place project.
+     - Run command ``git clone link_of_forked_repo``.
+     - Run Command ``cd kidney-exchnage ``.
+     - Open the folder wth your favourite text editor command.
```
Comment (priyanshuone6) [inline docs/newContributors.md:24]: The heading is smaller than the points, could you make the heading bigger in the whole document

**Comment by priyanshuone6** [comment]
Could you run the whole document through [Grammarly](https://grammarly.com) to fix potential spelling mistakes and grammar

**Comment by priyanshuone6** [comment]
Also, if you could add a prerequisite section in which maybe add a link to download git.
@siv2r thoughts?

**Comment by Sukriti-sood** [comment]
@priyanshuone6  I will do mentioned changes.

**Comment by Sukriti-sood** [comment]
@priyanshuone6  I have added changes , Still I didn't addede prerequisite section .

File: docs/newContributors.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+### 4.  So, Now you have an issue to work upon. So first you need is to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way:-
+
+<br>
+
+  ![](../images/Fork.png)
+
+<br></br>  
+
+### 5.  Clone the Repo you have forked:-
```
Comment (priyanshuone6) [inline docs/newContributors.md:33]: Could you please remove all the br tags and all the hashes in the numbered lists, thanks!

**Comment by Sukriti-sood** [comment]
@priyanshuone6  I  have added changes and please explain me briefly what to include in prerequisite section.

File: docs/newContributors.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+
+ 11.  Mentors will review your changes and will give feedback and if everything go well, your PR will get merged, Hurray  🎉🥳
+
+
+
+ 12.  So Now your PR Got Merged 🎉, Again Find or Create Issue and Continue the loop of Contributions to this Project
+
+
+<h1 align="center"> Happy Coding 💻</h1>
```
Comment (priyanshuone6) [inline docs/newContributors.md:82]: Please make it h3

**Comment by priyanshuone6** [comment]
Add git (with download link) and code editor in the prerequisites is all I can think of, feel free to add more

**Comment by siv2r** [comment]
> Also, if you could add a prerequisite section in which maybe add a link to download git.
> @siv2r thoughts?

The method to download git depends on the operating system used so, maybe a link to a youtube video explaining this will be useful I guess.

File: docs/newContributors.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+ 5.  Clone the Repo you have forked:-
+
+
+  
+      - Copy the link of the forked Repo.
+      
+      - Go to the desired location in the terminal, where you want to place the project.
+      
+      - Run command ``git clone link_of_forked_repo``.
```
Comment (siv2r) [inline docs/newContributors.md:39]: you can use `git clone https://github.com/<github username>/kidney-exchange.git` instead

**Review by siv2r — CHANGES_REQUESTED** [review]

File: docs/newContributors.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+ 6.  Set upstream to the Repo Using Command  `` git remote add upstream https://github.com/siv2r/kidney-exchange``.
+  
+
+
+ 7.  Make a branch using the command  ``git checkout -b branch_name ``.
+
+
+
+ 8.  Now it is the time to make required changes in the project 🌸
```
Comment (siv2r) [inline docs/newContributors.md:55]: Can you also add the following line?

You can look through the [good first issues](https://github.com/siv2r/kidney-exchange/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and select the one that you like to work on. If your interested issue is already assigned to someone, you can also collaborate to get it done. 

Also, add the following as point no 9.
If none of the issues matches your interest or things are a bit overwhelming, you can ask for guidance in the discord group.

Comment (Sukriti-sood) [reply docs/newContributors.md:55]: his should be 2or 3 point as I have included issues point ther

Comment (siv2r) [reply docs/newContributors.md:55]: What do u mean by issues points?

Comment (Sukriti-sood) [reply docs/newContributors.md:55]: @siv2r  I mean that  Points related to the issue are mentioned in Starting.

Comment (siv2r) [reply docs/newContributors.md:55]: Ahh, I get it now. Then can you add these to the start. 

>You can look through the good first issues and select the one that you like to work on. If your interested issue is already assigned to someone, you can also collaborate to get it done.

**Comment by siv2r** [comment]
The document is way better than I imagined. Keep working on it 💯 💯

File: docs/newContributors.md (hunk 7 above)
Comment (Sukriti-sood) [reply docs/newContributors.md:55]: Ook I will try do it by today, Sorry for being late.

Comment (siv2r) [reply docs/newContributors.md:55]: No worries, take your time. There is no time constrain here.

Comment (siv2r) [reply docs/newContributors.md:55]: Also, please click the button `Resolve conversation` after implementing the change. It helps to keep track of the progress.

**Comment by siv2r** [comment]
@Sukriti-sood The PR shows `17 files changed`. I think you committed the changes done by other contributors while updating your fork. I am not sure how to revert those commits. My suggestion would be to create a new PR (after synchronizing your fork with upstream) that contains only your changes.

**Pull request closed without merging**